service.toml

```diff
@@ -9,7 +10,7 @@ optional = ["object_mode"]
 optional = ["object_mode"]
 
 [namespace.storage.op.list]
-optional = ["list_mode"]
+optional = ["list_mode","continuation_token"]
```

Comment:
`continuation_token` support is optional, we can implement it in later PRs.

Reply:
Maybe ftp is not need token, just account?

Reply:
Oh, I get it.

Reply:
It can be used to list partially.

Reply:
Since ftp server has not its native token, now, the token is the records of index in iterator.